core/src/test/scala/com/microsoft/azure/synapse/ml/causal/VerifyDoubleMLEstimator.scala

```diff
@@ -92,6 +92,22 @@ class VerifyDoubleMLEstimator extends EstimatorFuzzing[DoubleMLEstimator] {
     assert(ateLow < ateHigh && ateLow > 0 && ateHigh < 150)
   }
 
+
+  test("Get individual treatment effect from transformer") {
+    val ldml = new DoubleMLEstimator()
+      .setTreatmentModel(new LogisticRegression())
+      .setTreatmentCol(mockLabelColumn)
+      .setOutcomeModel(new LinearRegression())
+      .setOutcomeCol("col2")
+      .setIteOutputCol("col2_ite")
+      .setIteStddevOutputCol("col2_ite_sd")
+      .setMaxIter(20)
+
+    val ldmlModel = ldml.fit(mockDataset)
+    val df = ldmlModel.transform(mockDataset)
```

Comment:
this test wont actually run any transform because of laziness, might want to add a quick sanity check on the output (Thats robust to noise so the test doesent fail)